File: UI/src/app/dtos/SaleryManagement/salaryPaymentDto.ts
import { BaseEntityDto } from "../base/baseEntityDto";

export class SalaryPaymentDto extends BaseEntityDto {
    employeeName: string | null;
    employeeId: number;
    basicSalary : number;
    tax : number;
    netAmount:number;
    date: Date;
    total:number;
    leave:number;
    deduction:number;
    lastName:string|null;
    allReadyPaid:string|null; 
}

File: API/Dtos/SalaryManagement/SalaryPaymentDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using API.Dtos.Base;

namespace API.Dtos.SalaryManagement
{
    public class SalaryPaymentDto: BaseEntityDto
    {
        public string ? EmployeeName{get;set;}
        
        public string? DepartmentName{get;set;}
        public int ? EmployeeId{get;set;}
        public double? BasicSalary {get;set;}

        public double? tax {get;set;}

        public double? NetAmount{get;set;}
        public DateTime Date{get;set;}
         public int ? Deduction{get;set;}
          public double? Total{get;set;}

          public int ? Leave{get;set;}

  
                   public string ? AllReadyPaid{get;set;} 

                   public string ? FiledNullMsg{get;set;} 
    }
}